area_printer.go

```diff
@@ -4,8 +4,13 @@ import (
 	"strings"
 
 	"atomicgo.dev/cursor"
+<<<<<<< HEAD
```

Comment:
It looks like the merge conflict message got left here. Just remove that and it should be good.